pkg/daemon/daemon.go

```diff
@@ -322,7 +327,16 @@ func (dn *Daemon) checkFiles(files []ignv2_2types.File) bool {
 // checkFileContents reads the file from the filepath and compares its contents
```

Comment:
should update doc comment with new name and additional info on file mode handling